go.mod

```diff
@@ -249,7 +251,7 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.14.8 => github.com/ethereum-optimism/op-geth v1.101408.0-rc.4.0.20240827042333-110c433a2469
+replace github.com/ethereum/go-ethereum v1.14.10 => github.com/0x00101010/op-geth v0.0.0-20241002184426-356ee643a541
```

Comment:
Can we just update this to point to the merge commit that landed on optimism branch: `d5a96613c22bc46238a21d6c0f805399c26c9d4c` please?

Reply:
Done! Thought we need to wait for a release first